.github/workflows/tests-gpu.yml

```diff
@@ -0,0 +1,22 @@
+name: GPU Tests
+
+on:
+  merge_group:
+
+jobs:
+  gpu-tests:
+
+    runs-on: ubuntu-22.04  # TODO replace with a GPU-enabled runner
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python 3.13
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.13
+    - run: pip install tox
+
+    - name: run Python tests
+      run: |
+        tox -e tests
```

Comment:
Do we also want to run architecture tests here?

Reply:
Yes, lets do it!